assembly/assembly-wsmaster-war/pom.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2012-2016 Codenvy, S.A.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+      Codenvy, S.A. - initial API and implementation
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>che-assembly-parent</artifactId>
+        <groupId>org.eclipse.che</groupId>
+        <version>4.3.0-RC1-SNAPSHOT</version>
+    </parent>
+    <artifactId>assembly-wsmaster-war</artifactId>
+    <packaging>war</packaging>
+    <name>Che IDE :: Compiling WS Master WAR</name>
```

Comment:
Is it compiling or packaging like https://github.com/eclipse/che/pull/1216/files#diff-d4c886c1cbe35ec2bcc892d158db1014R23 ?